e2e/testsuite/testsuite.go

```diff
@@ -143,13 +131,76 @@ func (s *E2ETestSuite) GetRelayerUsers(ctx context.Context, chainOpts ...ChainOp
 // with E2ETestSuite.StartRelayer if needed.
 // This should be called at the start of every test, unless fine grained control is required.
 func (s *E2ETestSuite) SetupChainsRelayerAndChannel(ctx context.Context, channelOpts func(*ibc.CreateChannelOptions), chainSpecOpts ...ChainOptionConfiguration) (ibc.Relayer, ibc.ChannelOutput) {
-	chainA, chainB := s.GetChains(chainSpecOpts...)
-	r := s.ConfigureRelayer(ctx, chainA, chainB, channelOpts)
-	chainAChannels, err := r.GetChannels(ctx, s.GetRelayerExecReporter(), chainA.Config().ChainID)
+	chains := s.GetAllChains(chainSpecOpts...)
+
+	r := relayer.New(s.T(), *LoadConfig().GetActiveRelayerConfig(), s.logger, s.DockerClient, s.network)
+
+	ic := s.newInterchain(ctx, r, chains, channelOpts)
+
+	buildOpts := interchaintest.InterchainBuildOptions{
+		TestName:  s.T().Name(),
+		Client:    s.DockerClient,
+		NetworkID: s.network,
+	}
+
+	s.Require().NoError(ic.Build(ctx, s.GetRelayerExecReporter(), buildOpts))
+
+	chainAChannels, err := r.GetChannels(ctx, s.GetRelayerExecReporter(), chains[0].Config().ChainID)
 	s.Require().NoError(err)
 	return r, chainAChannels[len(chainAChannels)-1]
 }
 
+// newInterchain constructs a new interchain instance that creates channels between the chains.
+func (s *E2ETestSuite) newInterchain(ctx context.Context, r ibc.Relayer, chains []ibc.Chain, channelOpts func(*ibc.CreateChannelOptions)) *interchaintest.Interchain {
+	channelOptions := ibc.DefaultChannelOpts()
+	// For now, set the version to the latest transfer module version
+	// DefaultChannelOpts uses V1 at the moment
+	channelOptions.Version = transfertypes.V2
```

Comment:
the previous code path handled backwards compatibility, this is still needed?

Reply:
good catch, will pull in those changes here